.rustfmt.toml

```diff
@@ -1,5 +0,0 @@
-# Doc: https://rust-lang.github.io/rustfmt/
-imports_granularity = "Crate"
```

Comment:
I find these grouping rules pretty helpful. Any ideas on keeping the functionalities after the change?

Reply:
It would be nice, but not possible on stable - would be pretty easy to write a custom linter, but probably not a priority right now

Reply:
(see https://github.com/ChainSafe/forest/pull/2876#discussion_r1194520439)